src/test/interface/example_output/mix.nom

```diff
@@ -7,12 +7,10 @@ No divergent transitions found.
 Checking E-BFMI - sampler transitions HMC potential energy.
 E-BFMI satisfactory.
 
-The following parameters had fewer than 0.001 effective draws per transition:
-  mu[1], mu[2], theta
-Such low values indicate that the effective sample size estimators may be biased high and actual performance may be substantially lower than quoted.
+Effective sample size satisfactory.
```

Comment:
It seems a touch concerning that something labeled as a problem before is now seen as okay -- is this really expected?

Reply:
yes, because rank-normalized split ESS estimates are higher than the old ESS estimates.

Reply:
I changed the direction of the comparison - now taking the lower value of ESS of bulk, tail.
the 3 parameters which were flagged before are now just above threshold - n_eff / num_samples is 0.002, but threshold is 0.001.    close, but no warning.